gammapy/cube/make.py

```diff
@@ -303,10 +325,47 @@ def _make_background(self):
 
         self.maps["background"] = background
 
+    def _make_edisp(self):
+        energy_axis = self.geom_true.get_axis_by_name("ENERGY")
+        if self.migra_axis is None:
+            axes = {
+                axis.name.lower(): axis for axis in self.observation.edisp.data.axes
+            }
+            self.migra_axis = axes["migra"]
+        geom_migra = self.geom_true.to_image().to_cube([self.migra_axis, energy_axis])
+        edisp_map = make_edisp_map(
+            edisp=self.observation.edisp,
+            pointing=self.observation.pointing_radec,
+            geom=geom_migra,
+            max_offset=self.offset_max,
+            exposure_map=self.maps["exposure_irf"],
+        )
+        self.maps["edisp"] = edisp_map
+
+    def _make_psf(self):
+        psf = self.observation.psf
+        if isinstance(psf, EnergyDependentMultiGaussPSF):
+            rad = np.linspace(0, 0.66, 67) * u.deg  # Arbitrary binning of 0.01 in rad
```

Comment:
@registerrier : Some `rad` values are still needed here. Put in an arbitrary binning here.

Reply:
One tiny last comment: maybe it's better to introduce the default directly in `EnergyDependentMultiGaussPSF.to_psf3d()`? Then you get rid of the extra code here and it becomes available for other use-cases as well...so you could put `.to_psf3d(rad=None)` and in `to_psf3d()` you put `if rad is None: ...`

Reply:
Done!